.rubocop_todo.yml

```diff
@@ -1,11 +1,23 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2019-12-19 16:15:54 -0500 using RuboCop version 0.76.0.
+# on 2020-01-21 15:56:08 -0500 using RuboCop version 0.76.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
 # Offense count: 2
+# Configuration parameters: CountComments.
+Metrics/ClassLength:
+  Max: 130
+
+# Offense count: 2
+# Cop supports --auto-correct.
+# Configuration parameters: AutoCorrect, AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
+# URISchemes: http, https
+Metrics/LineLength:
+  Max: 126
```

Comment:
this todo seems to be not required anymore, @cdmo could you also check?

Reply:
Nope.